core-web/libs/portlets/dot-ema/src/lib/shared/models.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+
+export enum MESSAGE_ACTIONS {
+    EDIT_CONTENTLET = 'edit-contentlet'
+}
+
+export enum CUSTOM_EVENTS {
+    EDIT_CONTENTLET_LOADED = 'edit-contentlet-loaded'
+}
```

Comment:
Is not clear what is the diff between these two and if I am creating a new one where it goes?

Reply:
Let's talk about this in the SU